packages/contracts-bedrock/test/kontrol/deployment/DeploymentSummary.t.sol

```diff
@@ -115,3 +127,133 @@ contract DeploymentSummary_TestL1CrossDomainMessenger is DeploymentSummary, L1Cr
     ///         L2CrossDomainMessenger, which is needed in this test
     function test_relayMessage_v2_reverts() external override { }
 }
+
+contract DeploymentSummary_TestL1ERC721Bridge is DeploymentSummary, L1ERC721Bridge_Test {
+    /// @notice super.setUp is not called on purpose
+    function setUp() public override {
+        // Recreate Deployment Summary state changes
+        DeploymentSummary deploymentSummary = new DeploymentSummary();
+        deploymentSummary.recreateDeployment();
+
+        // Set summary addresses
+        optimismPortal = OptimismPortal(payable(optimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(superchainConfigProxyAddress);
+        l2OutputOracle = L2OutputOracle(l2OutputOracleProxyAddress);
+        systemConfig = SystemConfig(systemConfigProxyAddress);
+        l1CrossDomainMessenger = L1CrossDomainMessenger(l1CrossDomainMessengerProxyAddress);
+        l1ERC721Bridge = L1ERC721Bridge(l1ERC721BridgeProxyAddress);
+
+        // Set up utilized addresses
+        alice = makeAddr("alice");
+        bob = makeAddr("bob");
+        vm.deal(alice, 10000 ether);
+        vm.deal(bob, 10000 ether);
+
+        // Bridge_Initializer setUp
+        L1Token = new ERC20("Native L1 Token", "L1T");
+
+        LegacyL2Token = new LegacyMintableERC20({
+            _l2Bridge: address(l2StandardBridge),
+            _l1Token: address(L1Token),
+            _name: string.concat("LegacyL2-", L1Token.name()),
+            _symbol: string.concat("LegacyL2-", L1Token.symbol())
+        });
+        vm.label(address(LegacyL2Token), "LegacyMintableERC20");
+
+        // Deploy the L2 ERC20 now
+        // L2Token = OptimismMintableERC20(
+        //     l2OptimismMintableERC20Factory.createStandardL2Token(
```

Comment:
@mds1 I've commented out what lines we're skipping from the `Bridge_Initializer.setUp`. Let me know if you prefer these comments be deleted

Reply:
These are commented out because we don't run them as part of deployment summary logic since they are needed? If so we can delete these commented out lines, and instead just have a comment at the top of setUp explaining why some setUp code was deleted

Reply:
Actually the current approach is consistent with what we do in e.g. `test_constructor_succeeds` so we can leave it as-is

Reply:
Sounds good to me! The rationale is that if someone wants to figure out what is not being executed, it should be easier with the commented code instead of just looking at what's being executed. However, if this doesn't seem right, I can change what we do in the other cases